src/pages/DetailTrip.js

```diff
@@ -22,6 +21,21 @@ export default function DetailTrip({
     onDeleteDestination(id);
     navigate(-1);
   }
+  const startDate = new Date(detailDestination[0].startDate);
+  const startYYYY = startDate.getFullYear();
+  let startMM = startDate.getMonth() + 1;
+  let startDD = startDate.getDate();
+  if (startDD < 10) startDD = '0' + startDD;
+  if (startMM < 10) startMM = '0' + startMM;
+  const displayStartDate = startDD + '.' + startMM + '.' + startYYYY;
+
+  const endDate = new Date(detailDestination[0].endDate);
+  const endYYYY = endDate.getFullYear();
+  let endMM = endDate.getMonth() + 1;
+  let endDD = endDate.getDate();
+  if (endDD < 10) endDD = '0' + endDD;
+  if (endMM < 10) endMM = '0' + endMM;
+  const displayEndDate = endDD + '.' + endMM + '.' + endYYYY;
```

Comment:
As you are using this more than once, maybe it is worth it extracting this into a custom hook (e.g. useDateFormation or something like this)